ui/src/serverIO.js

```diff
@@ -329,22 +327,12 @@ class ServerIO {
         }
 
         case 'loadReady': {
-          this.dispatch(
-            setLoading(false, 'SC Ready', record.message, true, () =>
-              this.dispatch(stopQueue()),
-            ),
-          );
-
+          this.dispatch(hideWaitDialog());
```

Comment:
Not sure about this to be honest. The purpose of the previous code is not clear: it hides the wait modal while still giving it a title, message and abort function. I traced it back to [here](https://github.com/mxcube/mxcubeweb/pull/597/files#diff-3ce702edf4adc9b09256a43ff92b1001d01e8ca66e6d547a52a587f4a5ffb580R175).